venues/ICLR.cc/2018/Conference/webfield/conferenceWebfield_alt.js

```diff
@@ -303,6 +307,45 @@ function renderContent(allNotes, submittedNotes, assignedNotePairs, userGroups,
     Webfield.setupAutoLoading(BLIND_INVITATION, PAGE_SIZE, submissionListOptions);
   }
 
+  // Withdrawn Papers tab
+  var withdrawnListOptions = _.assign({}, paperDisplayOptions, {
+    showTags: false,
+    container: '#withdrawn-papers'
+  });
+
+  if (withdrawnNotes.length) {
+    Webfield.ui.submissionList(withdrawnNotes, {
```

Comment:
I think this should use Webfield.ui.searchResults for this list like the My Submitted Papers tab on line 350.